src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/events.py

```diff
@@ -0,0 +1,41 @@
+"""
+Standardised class for pushing events from within the ADF Namespace
+"""
+
+
+import os
+import boto3
+
+
+class ADFEvents:
+    def __init__(
+        self, client: boto3.client, service, namespace="ADF", eventbus_arn=None
+    ) -> None:
+        """
+        Client: Any Boto3 EventBridge client
+        Service: The name of the Service e.g AccountManagement.EnableSupport
+        namespace: Defaults to ADF
+        eventbus_arn: Optionally specify a custom EventBridge ARN. If no ARN is specified, and no ENV variable set, will default to ADF-Event-Bus
+
+        """
+        self.events = client
```

Comment:
Thinking about making this optional. 
I.e if not supplied, it would make initialization nicer looking. 
```suggestion
        self.events = client if client else boto3.client("events", region_name=os.getenv("ADF_EVENTBUS_REGION", os.getenv("AWS_REGION")))
```